instagram/src/components/Authentication/Authenticate.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Login from '../Login/Login';
+
+const Authenticate = App => 
+    class extends React.Component{
+        constructor(props){
+            super(props);
+            this.state ={
+                logIn: false
+            };
+        }
+
+        componentDidMount(){
+            if(!localStorage.getItem('user')) {
+                this.setState({
+                    logIn: false
+                });
+            } else{
+                this.setState({
+                    logIn: true
+                });
+            }
+        }
+
+        render(){
+            if(this.state.logIn) return <App />;
```

Comment:
You can use a ternary statement on line 26 `{this.state.logIn ? <App /> : </Login}`